WordPress/Classes/ViewRelated/Post/Scheduling/SchedulingDatePickerViewController.swift

```diff
@@ -26,7 +26,7 @@ class DateCoordinator {
 // MARK: - Date Picker
 
 @available(iOS, introduced: 14.0)
```

Comment:
Perhaps this can be removed as well? I can think of arguments for or against, so either way is good with me, but just checking whether you've considered it.

Reply:
Good catch. I should have mentioned that I left that for #19709 . Thanks!